force-app/main/default/lwc/relationshipSettings/relationshipSettings.html

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="slds-grid--vertical slds-theme_default">
+        <div class="slds-col slds-p-top_large slds-p-left_small">
```

Comment:
I thought we dont need these classes in each single page?

Reply:
You're right! Removing it.